app/views/content/blog/becoming-a-mum-sparked-my-interest-in-teaching.md

```diff
@@ -30,4 +30,4 @@ Working with young people and helping them to understand the world around them i
 
 I now have job security, opportunities for progression, new skills and a postgraduate qualification. Not to mention the new car that isn't falling apart, a well-deserved holiday booked, paid-off credit cards and a little put away for a rainy day.
 
-We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply. [Find out more about how we can help](/help-and-advice).
+We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply. [Find out more about how we can help](/help-and-support).
```

Comment:
@EllieNodder 
I know this is the existing link in prod but the link text on this blog and all the other ones in the PR is not brilliant from an accessibility perspective - help with what? (Low priority change though)